src/gp-model/.gitignore

```diff
@@ -5,3 +5,5 @@ __pycache__/
 .vscode
 .DS_Store
 greater_kampala_metropolitan_area_kiidp_2012/
+shape_files/
```

Comment:
Is it possible to have the shape files stored in like google storage and just referenced in the code? It will make it more maintainability and easy to scale. Also makes the devops work abit easily. cc: @lilly-muyama

Reply:
@danielogen let me work on this in a subsequent PR